zcutil/build.sh

```diff
@@ -1,7 +1,30 @@
-#!/bin/bash
+#!/usr/bin/env bash
```

Comment:
What if env isn't there?  Also can you still pass build args like -j8 ? See comment here: http://unix.stackexchange.com/questions/29608/why-is-it-better-to-use-usr-bin-env-name-instead-of-path-to-name-as-my/29620#29620